jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwMouseInput.java

```diff
@@ -141,11 +144,14 @@ public GlfwMouseInput(final LwjglWindow context) {
     }
 
     private void onCursorPos(final long window, final double xpos, final double ypos) {
+        float[] xScale = new float[1];
+        float[] yScale = new float[1];
+        glfwGetWindowContentScale(window, xScale, yScale);
```

Comment:
And was it that the callback doesn't work? That need to call this on every cursor move? Maybe not a heavy call...